frontend/package.json

```diff
@@ -151,7 +151,12 @@
     "yaml-language-server": "0.5.4",
     "yup": "^0.27.0"
   },
+  "peerDependencies": {
+    "type-fest": "0.16.0"
+  },
```

Comment:
Why is this peer dependency here?

Reply:
from https://github.com/pmmmwh/react-refresh-webpack-plugin/blob/main/README.md
> Note 1: If you use webpack.config.ts, please also install type-fest as a peer dependency.

Reply:
But peer dependencies aren't installed by default.
If this is _required_ then it should be a dev dependency. But if everything works without it, then just this peer dependency.

Reply:
well everything works like this, also he mentions in the readme to install is as peer.

Reply:
as @christianvogt said, the peer dependencies are not installed. It seems to me that the README is just worded weirdly - `type-fest` is `peerDepedency` of `react-refresh-webpack-plugin` which means it should be in our `devDependencies`